### PR TITLE
Added StorInPickUp BE support

### PR DESCRIPTION
**Pull request by IvansZuks**

### CODE REVIEW

File: src/etc/schema.graphqls [hunk 1]
```diff
@@ -376,3 +383,30 @@ type AppliedTaxItemRate {
...
+type Store {
+    city: String
+    country: String
+    description: String
+    name: String
+    phone: String
+    pickup_location_code: String
+    postcode: String
+    region: String
+    street: String
```
Comment (alfredsgenkins) [inline src/etc/schema.graphqls:411]: Why to redeclare every field here, is not this an extension of that type?

**Review by alfredsgenkins — CHANGES_REQUESTED** [review]
Minor, otherwise good

File: src/etc/schema.graphqls (hunk 1 above)
Comment (IvansZuks) [reply src/etc/schema.graphqls:411]: We don't re-declare it. We declare it, because this is custom stuff (we don't extend already existing one)

**Review by yeegor — CHANGES_REQUESTED** [review]
You have committed a change to the permissions of this file: `src/Model/Resolver/GetStores.php`

This should not be done. In order to prevent such from happening, set `git config core.filemode false` either only for this repository or globally, on your machine.

**Comment by yeegor** [comment]
I assumed that this is not intentional. If this is intentional - please lmk y.

**Review by yeegor — APPROVED** [review]

**Pull request merged**